doc/api/assert.md

```diff
@@ -13,6 +13,71 @@ A `strict` and a `legacy` mode exist, while it is recommended to only use
 For more information about the used equality comparisons see
 [MDN's guide on equality comparisons and sameness][mdn-equality-guide].
 
+## Class: AssertionError
```

Comment:
As this is not global, should not this be `## Class: assert.AssertionError`?

Compare:
[`Class: net.Server`](https://nodejs.org/api/net.html#net_class_net_server) and [`Class: net.Socket`](https://nodejs.org/api/net.html#net_class_net_socket)